### DISCUSSION TITLE
How to type in EFB without conflict with the game

### DISCUSSION THREAD

**Discussion by TansyEliot** [flybywiresim/aircraft#3312] [Help] [answered]
Hi, I want to know how can I type in EFB without conflict with the game, for example, if I press B in EFB, the game adjust altimeter, if I press V in EFB, the game opens VFR map, how can I type without conflict?

**Comment by hotshotp** [comment, marked as answer]
unfortunately you can't - we're looking to fix that as its the most annoying issue

**Reply by TansyEliot** [reply]
A temporary fix is type your message in a notepad and paste in EFB.
